scripts/cli/build_cli_release.sh

```diff
@@ -8,6 +8,9 @@
 ###########################################
 
 # Note: This must be run from the root of the aptos-core repository
+
+set -e
```

Comment:
What is this for?

Reply:
This is to exit the bash script early when any of the following commands fail. If you don't provide this bash will execute all statements in the bash script, even if some of the preceding ones fail. This is btw a pretty common flag to put into any bash script.

Reply:
as a follow up. I think the:

```
EXIT_CODE=$?
if [ "$EXIT_CODE" != "0" ]; then
  echo "Build failed with exit code $EXIT_CODE"
  exit $EXIT_CODE
fi
```
part in your bash script is basically unnecessary when you do `set -e`.